packages/calypso-e2e/src/lib/pages/gutenberg-editor-page.ts

```diff
@@ -280,12 +293,44 @@ export class GutenbergEditorPage {
 	/**
 	 * Visits the published entry from the post-publish sidebar.
 	 *
-	 * @returns {Promise<void>} No return value.
+	 * @param options visit options
+	 * @param options.waitForNavigation allow navigation to finish before returning
 	 */
-	async _visitPublishedEntryFromPublishPane(): Promise< void > {
+	async _visitPublishedEntryFromPublishPane( {
+		waitForNavigation,
+	}: {
+		waitForNavigation: boolean;
+	} ): Promise< void > {
 		const frame = await this.getEditorFrame();
 
+		if ( ! waitForNavigation ) {
+			await frame.click( selectors.viewButton, { noWaitAfter: true } );
```

Comment:
Do we need to waitForLoadState?

Reply:
This branch only happens when someone has explicitly asked to not wait for the load state by passing the `waitForNavigation`.
No one's actually using this code path yet though. I only included in for completeness when I added the `waitForNavigation` flag.

Reply:
If the `waitForLoadState('networkidle')` call is extraneous in some instances, perhaps it should be downgraded to a `waitForLoadState('load')` or removed altogether. What is the situation requiring this flag?